packages/fluentui/accessibility/src/behaviors/Pill/pillBehavior.ts

```diff
@@ -19,6 +19,8 @@ export const pillBehavior: Accessibility<PillBehaviorProps> = p => ({
         performDismiss: {
           keyCombinations: [{ keyCode: keyboardKey.Delete }, { keyCode: keyboardKey.Backspace }],
         },
+      }),
+      ...(p.selectable && {
```

Comment:
should `actionable` also add the enter and space key handling ?

Reply:
I think yes as we also add role=buton in this case

Reply:
That makes sense, but I think we will miss a `dismissible` prop here then, WDYT?

Reply:
@ling1726 @jurokapsiar check what you think about https://github.com/microsoft/fluentui/pull/22839/commits/a08c6bfb97e95e40b6ee76fa85cf473e5307c8bc

Reply:
I am a little bit affraid of this as it is a big and hard to discover breaking change. I understand it would make sense to add dismissable, but do we need to change the meaning of actionable for that?

Reply:
looking at the whole PR I think it is ok. However the pill needs to be focusable also if it is dismissible - so that user can navigate to it before being able to delete